src/main/java/at/hannibal2/skyhanni/utils/ColorUtils.kt

```diff
@@ -8,4 +10,10 @@ object ColorUtils {
     fun getBlue(colour: Int) = colour and 0xFF
 
     fun getAlpha(colour: Int) = colour shr 24 and 0xFF
-}
\ No newline at end of file
+
+    fun blend(start: Color, end: Color, percent: Double) = Color(
+        (start.getRed() * (1 - percent) + end.getRed() * percent).toInt(),
+        (start.getGreen() * (1 - percent) + end.getGreen() * percent).toInt(),
+        (start.getBlue() * (1 - percent) + end.getBlue() * percent).toInt()
+    )
```

Comment:
Not sure if RGB is the best color space for blending. It is probably fine, but i think the function name should indicate that the blending is done in RGB.